src/js/views/home/View.jsx

```diff
@@ -27,6 +31,27 @@ const Home = () => {
   const history = useHistory();
   const classes = useStyles();
 
+  const devices = useSelector(devicesSelector);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(
+      deviceActions.getDevices({
+        page: {
+          number: 0 + 1,
+          size: 10,
```

Comment:
What if I have more than 10 favorite devices? 
What if I have 2 favorite devices but they are in page 4?
:thinking: